nsxt/data_source_nsxt_policy_project.go

```diff
@@ -0,0 +1,83 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	infra "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/orgs"
+)
+
+const defaultOrgID = "default"
+
+func dataSourceNsxtPolicyProject() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceNsxtPolicyProjectRead,
+
+		Schema: map[string]*schema.Schema{
+			"id":           getDataSourceIDSchema(),
+			"display_name": getDataSourceDisplayNameSchema(),
+			"description":  getDataSourceDescriptionSchema(),
+			"path":         getPathSchema(),
```

Comment:
We also need to expose associated edge clusters and T0, so that user could reference those in subsequent config